101-cloud-shell-vnet/azuredeploy.json

```diff
@@ -153,13 +149,12 @@
             "sku": {
                 "name": "Standard",
                 "tier": "Standard"
-            },
-            "properties": {}
+            }
         },
         {
             "type": "Microsoft.Relay/namespaces/providers/roleAssignments",
             "apiVersion": "2018-09-01-preview",
-            "name": "[concat(parameters('relayNamespaceName'), '/Microsoft.Authorization/', guid(subscription().subscriptionId, parameters('relayNamespaceName')))]",
+            "name": "[concat(parameters('relayNamespaceName'), '/Microsoft.Authorization/', guid(variables('contributorRoleId'), parameters('azureContainerInstanceOID'), parameters('relayNamespaceName')))]",
```

Comment:
for the guid use:

guid(variables('contributorRoleId'), parameters('azureContainerInstanceOID'), variables('relayNamespaceId'))